fiat-roles/src/main/java/com/netflix/spinnaker/fiat/permissions/RedisPermissionsRepository.java

```diff
@@ -398,47 +389,19 @@ private UserPermission getUserPermission(String userId, RawUserPermission raw) {
     return permission;
   }
 
-  private Table<String, ResourceType, Response<Map<String, String>>> getAllFromRedis() {
-    try {
-      Set<String> allUsers = scanSet(allUsersKey());
-      return getAllFromRedis(allUsers);
-    } catch (Exception e) {
-      log.error("Storage exception reading all entries.", e);
-      return null;
-    }
-  }
-
-  private Table<String, ResourceType, Response<Map<String, String>>> getAllFromRedis(
-      Set<String> userIds) {
-    if (userIds.size() == 0) {
-      return HashBasedTable.create();
-    }
-
-    try {
-      final Table<String, ResourceType, Response<Map<String, String>>> responseTable =
-          ArrayTable.create(
-              userIds,
-              new ArrayIterator<>(
-                  resources.stream().map(Resource::getResourceType).toArray(ResourceType[]::new)));
-      for (List<String> userIdSubset : Lists.partition(new ArrayList<>(userIds), 10)) {
-        redisClientDelegate.withMultiKeyPipeline(
-            p -> {
-              for (String userId : userIdSubset) {
-                resources.stream()
-                    .map(Resource::getResourceType)
-                    .forEach(
-                        r -> {
-                          responseTable.put(userId, r, p.hgetAll(userKey(userId, r)));
-                        });
-              }
-              p.sync();
-            });
-      }
-      return responseTable;
-    } catch (Exception e) {
-      log.error("Storage exception reading all entries.", e);
-    }
-    return null;
+  private RawUserPermission getRawUserPermissionsFromRedis(String userId) {
```

Comment:
There's a lot of overlap between this and getFromRedis that it would be nice to consolidate